src/test/java/com/marizueva/laboratory/hw4/testdata/UsedInTestTerms.java

```diff
@@ -0,0 +1,20 @@
+package com.marizueva.laboratory.hw4.testdata;
+
+public class UsedInTestTerms {
+    public static final String homePageTitle = "Home Page";
+    public static final String loggedUserName = "ROMAN IOVLEV";
+    public static final String homeEpamFrameworkTitle = "EPAM FRAMEWORK WISHES";
+    public static final String homeLoremTitle = "LOREM IPSUM";
+    public static final String diffElSelenLog =
+            "14:29:07 metal: value changed to  Selen".substring(9);
+    public static final String diffElDropdownYellowLog =
+            "14:30:16 Colors: value changed to Yellow".substring(9);
+    public static final String diffElWindFalseLog =
+            "14:57:06  Wind: condition changed to false".substring(9);
+    public static final String diffElWaterFalseLog =
+            "14:57:05  Water: condition changed to false".substring(9);
+    public static final String diffElWindTrueLog =
+            "14:57:06  Wind: condition changed to true".substring(9);
+    public static final String diffElWaterTrueLog =
+            "14:57:05  Water: condition changed to true".substring(9);
```

Comment:
I guess that you could delete time from the text. This is allow you avoid getting substring from it

Reply:
fixed